ui-components/src/notifications/AlertBox.scss

```diff
@@ -1,86 +1,108 @@
 .alert-box {
-  @apply relative;
-  @apply py-3;
-  @apply px-4;
-  @apply leading-snug;
-  @apply flex;
-  @apply items-center;
+  --vertical-padding: var(--bloom-s3);
+  --horizontal-padding: var(--bloom-s4);
+  --background-color: var(--bloom-color-primary-light);
+  --alert-background-color: var(--bloom-color-alert-light);
+  --alert-invert-background-color: var(--bloom-color-alert);
+  --notice-background-color: var(--bloom-color-primary-light);
+  --notice-invert-background-color: var(--bloom-color-primary);
+  --success-background-color: var(--bloom-color-success-light);
+  --success-invert-background-color: var(--bloom-color-success);
+  --warn-background-color: var(--bloom-color-warn-light);
+  --warn-invert-background-color: var(--bloom-color-warn);
+  --text-color: var(--bloom-color-gray-900);
+  --close-icon-color: var(--bloom-color-gray-900);
+  --font-weight: 600;
+  --max-width: var(--bloom-width-5xl);
+  --line-height: 1.375;
+
+  padding: var(--vertical-padding) var(--horizontal-padding);
+  background-color: var(--background-color);
+  color: var(--text-color);
+  font-weight: var(--font-weight);
+  position: relative;
+  display: flex;
+  align-items: center;
+  line-height: var(--line-height);
 
   .alert-box_inner {
-    @apply m-auto;
-    @apply max-w-5xl;
-    @apply px-4;
-    @apply flex-1;
-    @apply flex;
-    @apply items-center;
+    margin: auto;
+    max-width: var(--max-width);
+    padding-right: var(--bloom-s4);
+    padding-left: var(--bloom-s4);
```

Comment:
should these be the --horizontal-padding?

Reply:
Actually this is unrelated to this PR, but the `boundToLayoutWidth` seems to double the padding and don't know if that was originally intended. Definitely not something that needs to be tackled in this refactor though, just curious if we know if that was intended

Reply:
Updated to use the var here! And in terms of the boundToLayoutWidth I think that was _maybe_ intentional - we only seem to use that for the alert on the listing preview that spans the whole page, and we did have to update some vertical padding so the text would fit around the site logo - @jaredcwhite do you remember if the horizontal padding was the same?